### DISCUSSION TITLE
Missing Image file

### DISCUSSION THREAD

**Discussion by Craythorneuk** [1000001101000/Debian_on_Buffalo#209] [Q&A]
I am trying to install Debian Bookworm on an LS210D. I am looking in the Debian-12.4.0-armhf-netinst.iso CD image for the installation files. In the install.ahf folder I can find several initrd.gz files but no uImage files. Am I looking in the wrong place?
My eventual objective is to install rsync server on the Buffalo to replace a WD MyBookLive which has bricked.

**Comment by 1000001101000** [comment]
The installer files are contained within the project. Take a look under Bookworm->installer_images->armhf_devices

**Reply by Craythorneuk** [reply]
Many thanks. All good now. I must have come across as rather thick!
